app/uk/gov/hmrc/exports/services/mapping/governmentagencygoodsitem/StatisticalValueAmountBuilder.scala

```diff
@@ -31,18 +31,14 @@ class StatisticalValueAmountBuilder @Inject()() extends ModifyingBuilder[ExportI
   def buildThenAdd(exportItem: ExportItem, wcoGovernmentAgencyGoodsItem: WCOGovernmentAgencyGoodsItem): Unit =
     exportItem.itemType.foreach { itemType =>
       wcoGovernmentAgencyGoodsItem.setStatisticalValueAmount(
-        createWCODecStatisticalValueAmount(Some(BigDecimal(itemType.statisticalValue)), Some(defaultCurrencyCode))
+        createWCODecStatisticalValueAmount(itemType.statisticalValue, defaultCurrencyCode)
       )
     }
 
```

Comment:
Got rig of the useless string -> optional scala BD -> non-optional java BD conversions here